client/app/loginScreen.tsx

```diff
@@ -1,29 +1,87 @@
 import { useDataContext } from '@/context/globalContext';
 import { getUser } from '@/services/apiService';
-import { Link } from 'expo-router';
+import { UserType } from '@/types';
+import { Link, router } from 'expo-router';
 import { useEffect } from 'react';
-import { View, Text, Pressable } from 'react-native';
-
+import { Controller, useForm } from 'react-hook-form';
+import { View, Text, Pressable, TextInput } from 'react-native';
 
 export default function loginScreen() {
+  const {
+    control,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<Partial<UserType>>();
+
   const { setUser } = useDataContext();
 
   useEffect(() => {
     async function setup() {
       const user = await getUser(1);
       setUser(user);
-
     }
     setup();
-  }, [])
+  }, []);
+
+  const onSubmit = (data: Partial<UserType>) => {
+    console.log(data);
+    router.replace('/profilesScreen');
+  };
+
   return (
-    <View>
-      {/*  //TODO: review asChild & Pressable*/}
-      <Link href='/profilesScreen' asChild>
-        <Pressable>
-          <Text>Login Page</Text>
-        </Pressable>
-      </Link>
-    </View>
+    <>
+      <View className='p-8 flex flex-col flex-1 bg-dark-grey'>
+        {
+          <>
+            <Text className='text-4xl font-extrabold text-green mb-10 text-center'>magicInk</Text>
+            <Text className='text-2xl font-extrabold text-white text-center'>Enter the wonderful world of magic!</Text>
```

Comment:
I would change it to something different:

- MagicInk: "Step into a realm of endless adventures!"
- MagicInk: "Unlock the magic in every story!"
- MagicInk: "Discover the magic within each tale!"
- MagicInk: "Dive into enchanting narratives!"
- MagicInk: "Experience the wonder of MagicInk tales!"
- MagicInk: "Embark on an unforgettable journey!"
- MagicInk: "Uncover the secrets of fantastical worlds!"
- MagicInk: "Join the adventure in every story!"
- MagicInk: "Let your imagination soar with MagicInk!"
- MagicInk: "Find your next magical adventure here!"

Reply:
sweet thanks - i like the first one